### DISCUSSION TITLE
poly disperse - Custom

### DISCUSSION THREAD

**Discussion by joker-ccm** [VirtualPhotonics/MieSimulatorGUI#10] [Q&A] [answered]
What format is the file for loading custom data or what are the requirements?

**Comment by janakarana** [comment, marked as answer]
All input files are in text (.txt) format. Sample input files are provided in the [CustomDataSamples](https://github.com/VirtualPhotonics/MieSimulatorGUI/tree/master/CustomDataSamples) folder
